sdk/include/ndk/pstypes.h

```diff
@@ -906,7 +906,7 @@ typedef struct _PROCESS_PRIORITY_CLASS
 {
     BOOLEAN Foreground;
     UCHAR PriorityClass;
-} PROCESS_PRIORITY_CLASS, *PPROCESS_PRIORITY_CLASS;
+}__attribute__((aligned(sizeof(ULONG)))) PROCESS_PRIORITY_CLASS, *PPROCESS_PRIORITY_CLASS;
 
 typedef struct _PROCESS_FOREGROUND_BACKGROUND
```

Comment:
This structure likely has the same problem. You could look at that in a follow-up if you wanted.

Reply:
Sure thing, I can take a look at this structure as well. But I'd rather create a separate PR for that to avoid cluttering this one with unrelated changes.

Reply:
That's fine, I think